packages/editor/src/components/preview-dropdown/index.js

```diff
@@ -44,6 +45,12 @@ export default function PreviewDropdown( { forceIsAutosaveable, disabled } ) {
 			};
 		}, [] );
 	const { setDeviceType } = useDispatch( editorStore );
+	const { __unstableSetEditorMode } = useDispatch( blockEditorStore );
+
+	const handleDevicePreviewChange = ( newDeviceType ) => {
```

Comment:
Should we add doc comment for the new function?
This is what I was thinking:
```
/**
* Handles the selection of a device type from preview dropdown.
*
* Turns off the zoom out mode after setting the device type.
*
* @param {string} newDeviceType The selected device type.
*/
```

Reply:
@AhmarZaidi 

I think there are no clear rules about JSDoc, but this is an internal function that is not exposed, and the process it performs is just two lines.

I personally don't think JSDoc is necessary because we can see what it's doing at a glance and I feel like JSDoc is a bit over the top. 😄